generate_outpatients.py

```diff
@@ -125,12 +125,15 @@
     )
     .withColumn(
         "type",
-        F.concat(
-            F.when(F.col("is_adult"), "adult").otherwise("child"),
-            F.lit("_"),
-            F.when(F.col("is_surgical_specialty"), "surgical").otherwise(
-                "non-surgical"
-            ),
+        F.when(F.col("tretspef").isin(["424", "501", "505", "560"]), "maternity")
```

Comment:
This is now creating maternity, {adult, child}-{surgical,non-surgical}

Do we not want to maintain the grouping, so this maternity spec should happen in same grouping as the surgical/non surgical?

It's arguable that we wouldn't want the difference in ages in maternity though.

Reply:
Have checked with @swyatt-SU  and @jakep03  and we're ok to go with  maternity, {adult, child}-{surgical,non-surgical}